### PR TITLE
Created LandingPage Container,Routing for LandingPage and Store was Provided

### PR DESCRIPTION
**Pull request by kishore0079**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -53,7 +53,8 @@
     "eslint-plugin-prettier": "^3.1.4",
     "eslint-plugin-react": "^7.21.5",
     "eslint-plugin-react-hooks": "^4.2.0",
-    "react-test-renderer": "^17.0.1"
+    "react-test-renderer": "^17.0.1",
```
Comment (pranavakkian) [inline package.json]: remove this

File: src/Components/LandingPage/index.jsx [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import { useSelector } from 'react-redux';
+// import { Header } from 'semantic-ui-react';
+
+function LandingPage() {
+  const userName = useSelector((state) => state.UserInfoReducer.userName);
+  const display = `Welcome ${userName}`;
```
Comment (pranavakkian) [inline src/Components/LandingPage/index.jsx]: why this variable?

File: src/RootPath.js [hunk 3]
```diff
@@ -0,0 +1,2 @@
+const RootPath = '/';
```
Comment (pranavakkian) [inline src/RootPath.js]: rename this file constants.js OR routePaths.js because u will be adding all the paths in this file only.

**Review by pranavakkian — CHANGES_REQUESTED** [review]

File: src/Components/LandingPage/index.jsx [hunk 4]
```diff
@@ -0,0 +1,13 @@
+import { useSelector } from 'react-redux';
+// import { Header } from 'semantic-ui-react';
+
+function LandingPage() {
+  const userName = useSelector((state) => state.UserInfoReducer.userName);
+  return (
+    <div>
+      <h1>{`Welcome ${userName}`}</h1>
```
Comment (pranavakkian) [inline src/Components/LandingPage/index.jsx]:
```suggestion
      <h1>Welcome {userName}</h1>
```

File: src/Components/LandingPage/index.jsx [hunk 5]
```diff
@@ -0,0 +1,13 @@
+import { useSelector } from 'react-redux';
+// import { Header } from 'semantic-ui-react';
```
Comment (pranavakkian) [inline src/Components/LandingPage/index.jsx]: Avoid comments.
```suggestion
```

File: src/Constants.js [hunk 6]
```diff
@@ -0,0 +1,2 @@
+const RootPath = '/';
```
Comment (pranavakkian) [inline src/Constants.js]: keep this file inside a folder named constants

**Review by pranavakkian — CHANGES_REQUESTED** [review]

**Review by pranavakkian — APPROVED** [review]

**Review by shivalibaghel7 — APPROVED** [review]

**Pull request merged**